public/static/locales/fr/admin-templates.json

```diff
@@ -16,8 +16,14 @@
     "delete-check": "Êtes-vous sûr de vouloir supprimer ce formulaire ?",
     "confirm-delete": "Confirmer Supprimer",
     "new": "Le formulaire a été créé avec succès",
-    "formID": "Form ID:",
-    "bearerToken": "Bearer Token:",
-    "edit": "Edit Form Config File:"
+    "formID": "ID de formulaire :",
```

Comment:
Could be "Identifiant du formulaire" in French

Reply:
I copied that from the same file... I'll change it to your suggestion in both places.